Add second search to certification filter

josephsnyder opened the pull request:
Add another ajax search to the certification filter to check for submissions
that have previous revisions that have been certified.

It does this by adding a second database call which searches through each revision of a submission for a matching certification level.

josephsnyder committed:
Add second search to certification filter

Add another ajax search to the certification filter to check for submissions
that have previous revisions that have been certified.

BetsyMcPhail commented:
Is it possible to rename 'revisionID' to something else? It's a little confusing with both a 'revisionId' and a 'revisionID' in the same structure (see function ajaxSearch() in index.index.js).

Otherwise, looks good to me.

josephsnyder committed:
Rename variable used to store past revision number.

Rename the revisionID variable to pastCertificationID to eliminate
confusion about the interaction between that and the sorting by
revisionid.
